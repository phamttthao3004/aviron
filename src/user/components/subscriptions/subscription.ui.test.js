import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import SubscriptionUI from './subscription.ui';

describe('Subscription UI', () => {
    it('Render Subscription UI component', () => {
        const props = { data: undefined, state: undefined, onChange: undefined, onSearch: undefined, setOpenAdd: undefined, setOpenCancel: undefined };
        const com = shallow(<SubscriptionUI {...props} />);
        expect(shallowToJson(com)).toMatchSnapshot();
    });
});
