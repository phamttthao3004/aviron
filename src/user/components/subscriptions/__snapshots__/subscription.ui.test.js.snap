// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Subscription UI Render Subscription UI component 1`] = `
<div
  id="subscription-page"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="warning"
      >
        <div>
          <i
            className="icon-ic_warning_24px"
          />
          <span>
            Transaction expired: over 10 munites for transaction
          </span>
        </div>
      </div>
      <CmsTable
        className=""
        data={Array []}
        mode="edit"
        onChange={[Function]}
        onSearch={[Function]}
        total={0}
      >
        <div
          className="cms-table-content"
        >
          <table
            className="table aviron-table action-table call-data-table"
          >
            <thead>
              <tr>
                <th>
                  #
                </th>
                <th>
                  Aviron key
                </th>
                <th>
                  Subscription type
                </th>
                <th>
                  Machine name
                </th>
                <th>
                  Machine id
                </th>
                <th>
                  Activated
                </th>
                <th>
                  Status
                </th>
                <th>
                  Next/Last bill date
                </th>
              </tr>
            </thead>
            <tbody />
          </table>
        </div>
      </CmsTable>
      <SubscriptionAddDialogUI
        setOpenAdd={[Function]}
        state={Object {}}
      />
      <SubscriptionCancelDialogUI
        setOpenCancel={[Function]}
        state={Object {}}
      />
    </div>
  </div>
</div>
`;
